Resources/Prototypes/Entities/Objects/Tools/jetpacks.yml

```diff
@@ -48,7 +48,6 @@
       quickEquip: false
       slots:
         - Back
-        - suitStorage
```

Comment:
wrong pr? this already exists in #14187

Reply:
Woah did I do that? totally unintentional.  I'll fix that immediately.